### PR TITLE
Bra 14 homework 01

### PR DESCRIPTION
**Pull request by dev-kucha**
Lesson 14
Part 1

### CODE REVIEW

File: common.blocks/blog-page/css/stiles.css [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    font-size: 32px;
+    font-weight: 700;
+    -webkit-transform: translateY(-4px);
+        -ms-transform: translateY(-4px);
+            transform: translateY(-4px);
+}
+
+.post__data-month {
+    font-size: 21px;
+    font-weight: 400;
```
Comment (roskyba) [inline common.blocks/blog-page/css/stiles.css]: 400 is a `normal` value

File: common.blocks/header/logo/styles.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Browsers: last 4 version
+*/
+
+@import url('https://fonts.googleapis.com/css2?family=Pacifico&display=swap');
+
+.logo {
+    color: #262626;
+    font-family: Pacifico;
+    font-size: 50px;
+    font-weight: 400;
```
Comment (roskyba) [inline common.blocks/header/logo/styles.css]: 400 is the default value, like `normal`

File: common.blocks/footer/footer.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            justify-content: space-between;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+}
+
+.footer__logo {
+    color: #b9b9b9;
+    font-size: 25px;
+    font-weight: 400;
```
Comment (roskyba) [inline common.blocks/footer/footer.css]: `normal` value

File: common.blocks/blog-page/css/stiles.css [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+    }
+}
+    
+@media screen and (max-width:768px) {
+    .post-2 {
+        display: none;
+    }
+}
+    
+@media screen and (max-width:320px) {
```
Comment (roskyba) [inline common.blocks/blog-page/css/stiles.css]: < 320px

File: common.blocks/contact-page/css/stiles.css [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+        font-size: 14px;
+    }
+
+    .address-block__dd {
+        color: #262626;
+        font-size: 16px;
+    }
+}
+    
+@media screen and (max-width:320px) {
```
Comment (roskyba) [inline common.blocks/contact-page/css/stiles.css]: < 320px

File: common.blocks/gallary/css/styles.css [hunk 6]
```diff
@@ -0,0 +1,241 @@
+/*
+* Prefixed by https://autoprefixer.github.io
+* PostCSS: v7.0.29,
+* Autoprefixer: v9.7.6
+* Browsers: last 4 version
+*/
+
+.gallary {
```
Comment (roskyba) [inline common.blocks/gallary/css/styles.css]: gallary ===> gallery

File: common.blocks/gallary/css/styles.css [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+    .pic-4 {
+        margin-right: 20px;
+    }
+    
+    .gallary-row-2 {
+        height: 200px;
+    }
+}
+    
+@media screen and (max-width:320px) {
```
Comment (roskyba) [inline common.blocks/gallary/css/styles.css]: < 320px

File: common.blocks/main/about-us/styles.css [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+     .about-us__content {
+        text-align: center;
+    }
+
+    .about-us {
+        height: 50vh;
+    }
+}
+    
+@media screen and (max-width:320px) {
```
Comment (roskyba) [inline common.blocks/main/about-us/styles.css]: < 320px

File: common.blocks/our-services/css/styles.css [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+    .services-item__text {
+        width: 100%;
+    }
+
+    .services-item__pic::before {
+        font-size: 30px;
+    }
+}
+    
+@media screen and (max-width:320px) {
```
Comment (roskyba) [inline common.blocks/our-services/css/styles.css]: < 320px

File: common.blocks/main/get/styles.css [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+        background-image:  -o-linear-gradient(
+            transparent,
+            rgba(255, 255, 255, 0.8), transparent), url(./img.webp);
+        background-image:  linear-gradient(
+            transparent,
+            rgba(255, 255, 255, 0.8), transparent), url(./img.webp);
+    }
+}
+    
+@media screen and (max-width:320px) {
```
Comment (roskyba) [inline common.blocks/main/get/styles.css]: < 320px

**Comment by dev-kucha** [comment]
All found bugs are fixed 22-03-2021 20:24

**Review by roskyba — APPROVED** [review]

**Comment by dev-kucha** [comment]
Done

**Pull request closed without merging**